sdk/core/azure-core/inc/azure/core/url.hpp

```diff
@@ -243,6 +243,13 @@ namespace Azure { namespace Core {
      */
     std::string GetUrlWithoutQuery() const { return GetUrlWithoutQuery(false); }
 
+    /**
+     * @brief Get Scheme, host and port.
+     *
+     * @return Url authority.
+     */
+    std::string GetUrlAuthorityWithScheme() const;
```

Comment:
Can this be private for now?

Reply:
Needs to be accessed by KeyVault, we would need friend-access for it if we move to private